packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -2675,3 +2712,26 @@ typedef SelectableRegionContextMenuBuilder = Widget Function(
   BuildContext context,
   SelectableRegionState selectableRegionState,
 );
+
+/// A controller for a [SelectionArea] or [SelectableRegion].
+///
+/// Utilize this controller to programatically clear the content of a
+/// [SelectionArea] or [SelectableRegion], or select all the contents under it.
+class SelectionController with ChangeNotifier {
```

Comment:
Looks like their is a name conflict with `SelectionController`, this needs a different name.

Reply:
Ideas:

 * TextSelectionController
 * SelectionAreaController